README.md

```diff
@@ -93,14 +99,19 @@ This URL can be used in Session to join the group
 
 ### Step 6: Add moderator to room
 
-in the root directory run 
+in the root directory run
 
-```python3 -msogs --rooms ROOMNAME --add-moderators SESSIONID```
+```bash
+python3 -msogs --rooms ROOMNAME --add-moderators SESSIONID
+```
 
-for example 
+for example
 
-```python3 -msogs --rooms fishing --add-moderators 05d871fc80ca007eed9b2f4df72853e2a2d5465a92fcb1889fb5c84aa2833b3b40```
+```bash
+python3 -msogs --rooms fishing --add-moderators 05d871fc80ca007eed9b2f4df72853e2a2d5465a92fcb1889fb5c84aa2833b3b40
+```
```

Comment:
Should also say something like: "Run `python3 -msogs --help` to see all available sogs command-line options."

Reply:
Perhaps:

"You can list all rooms and moderators/admins using `python3 -msogs -Lv`.  Run `python3 -msogs --help` to see all available command-line options."